src/id_mapper/graph.py

```diff
@@ -13,18 +13,37 @@
 # limitations under the License.
 
 import logging
+import os
+import time
 
-from py2neo import Node, Relationship
+from py2neo import Graph, Node, Relationship
+from py2neo.packages.httpstream.http import SocketError
 
 
 logger = logging.getLogger(__name__)
 
+connected = False
+while not connected:
+    try:
+        graph = Graph(
```

Comment:
If graph is now defined globally  for the module, perhaps it's nice to mark it in all caps which is what we do in memote OR probably better to follow PEP8 (https://www.python.org/dev/peps/pep-0008/#global-variable-names) and prefix them with an underscore?